src/udiskslinuxfilesystem.c

```diff
@@ -1284,6 +1284,7 @@ is_system_managed (UDisksDaemon *daemon,
 
 static void trigger_mpoint_cleanup (const gchar *mount_point)
 {
+#ifdef HAVE_LIBSYSTEMD_LOGIN
```

Comment:
You can remove this commit -- this function is being removed in 8f89e61636579c6284ba58439493a9c32f5caa97

Reply:
Aaa, busted. Wanted to boost my commit stats a little bit...